cmake/iree_aie_xrt.cmake

```diff
@@ -0,0 +1,206 @@
+# Copyright 2024 The IREE Authors
+#
+# Licensed under the Apache License v2.0 with LLVM Exceptions.
+# See https://llvm.org/LICENSE.txt for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+if(TARGET iree-aie-xclbinutil)
+  return()
+endif()
+
+include(${CMAKE_CURRENT_LIST_DIR}/iree_aie_utils.cmake)
+
+# ##############################################################################
+# boost
+# ##############################################################################
```

Comment:
Bruh lol

![image](https://github.com/user-attachments/assets/736ed4d9-ca7d-4669-adad-42993a9eb2ee)

The dep on boost is through XRT and it's been a source of pain since the beginning. If we could avoid it we would (and maybe we will in the near future...).

Reply:
Fair enough. At least putting my name on the record supporting dropping the dep somehow or isolating it as much as possible :P